compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +166,55 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init(true);
+
+  // check if addV2 is Custom before this pass is executed.
+  auto addV2 = dynamic_cast<luci::CircleCustom *>(_g.addV2());
+  EXPECT_NE(nullptr, addV2);
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  // check if Custom(AddV2) is converted to Add after this pass is executed.
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+}
```

Comment:
@seanshpark @mhs4670go 

After the offline discussion, 
I modified the code so that this can test the main features of the pass (`Custom(AddV2)`->`Add`).

1/ if pass is not executed, AddV2 is Custom
2/ if pass is executed, Custom(AddV2) will be converted to Add.

Could you reivew this PR again? :)